index.html

```diff
@@ -36,3 +39,134 @@ <h2 class="support-text">I’am a Software Developer</h2>
             <li class="links"><a href="#"><img src="images/Vector3.svg" alt="Media"></a></li>
         </ul>
     </section>
+    <section class="grid-container">
+        <h2 class="title">My Recent Works</h2>
+        <div class="image-line1"><img src="images/line1.svg" alt=""></div>
+        <div class="card-1">
+            <div class="image"><img src="images/imagecard1.png" alt=""></div>
+            <h3 class="multipost-h">Multi-Post Stories</h3>
+            <p class="para1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="tags">
+                <li class="csbox">CSS</li>
+                <li class="htmlbx">HTML</li>
+                <li class="bootbox">Bootstrap</li>
+                <li class="rubox">Ruby</li>
+            </ul>
+            <button class="button1">See Project</button>
+        </div>
```

Comment:
- [ ] According to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins). So Kindly, remove this `margin-left` from this card to look similar to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

| Your Design  | Figma Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/176979518-25d7342f-26a6-4e2f-971f-4ac3348a92b0.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/176979569-ab873405-bedb-4644-bfa9-873135be245b.png" width="500" /> |

> To do this I suggest removing `align-items: center;` from `.card-1`, removing the inline margin from the button, and adding fixed `width` to the button.